### PR TITLE
LOOFAH-32 Create project to seed database

### PR DESCRIPTION
**Pull request by elishkaflint**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM mongo
+
+WORKDIR /usr/src/app
+
+COPY . .
+
+RUN chmod +x ./entrypoint.sh
+RUN apt-get update && apt-get install -y ruby
```
Comment (Learner5200) [inline Dockerfile]: Nitpick: might be an idea to put this earlier on in the dockerfile, so we can take advantage of the cache for this step. https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#use-multi-stage-builds has more context

Comment (elishkaflint) [reply Dockerfile]: thanks for the link - done!

File: entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+gem install csv
```
Comment (Learner5200) [inline entrypoint.sh:3]: Nitpick: in the same vein as my other comment, we could move these `gem install`s to the dockerfile (just after installing ruby), so they get cached.

**Review by Learner5200 — APPROVED** [review]

File: entrypoint.sh (hunk 2 above)
Comment (elishkaflint) [reply entrypoint.sh:3]: can't immediately work out why i get an error with the json gem when i move this to the dockerfile. gonna leave as is for now and add it to my tech debt list 😕

**Pull request merged**